client/src/components/Overview/ImageGallery.jsx

```diff
@@ -1,7 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Carousel from 'App/Carousel.jsx';
 
-function ImageGallery() {
-  return null;
+function ImageGallery({ photos }) {
+  // TODO: create photo_ids for photos
+  const [mainImageIndex, setMainImageIndex] = useState(0);
+
+  const handleThumbnailClick = (index) => {
+    setMainImageIndex(index);
+  };
+
+  const images = photos.map((photo, i) => (
+    <img
+      key={i}
```

Comment:
One thing that's not a big deal but try to avoid duplicate keys/using index for a key when possible

Reply:
Yes, good point. I've augmented the photo objects with an id now.